tests/php/WordPressThemeTest.php

```diff
@@ -25,6 +40,10 @@ public function testSinglePage(): void
     {
         $html = $this->render('single.php', 1);
 
+        $this->assertStringContainsString(
+            sprintf('<body data-amo-base-url="%s">', self::$AMO_BASE_URL),
```

Comment:
I think you can do:

```suggestion
            "<body data-amo-base-url=\"{self::$AMO_BASE_URL}\">",
```

Reply:
That doesn't seem to work, but `"<body data-amo-base-url=\"$AMO_BASE_URL\">"` does, but only after I copy `self::$AMO_BASE_URL` into `$AMO_BASE_URL`. I couldn't seem to figure out how to use `self::$AMO_BASE_URL` in a template string. I'll update the patch with the change I just described.